shared/source/os_interface/linux/xe/ioctl_helper_xe.cpp

```diff
@@ -1317,6 +1320,37 @@ int IoctlHelperXe::xeVmBind(const VmBindParams &vmBindParams, bool isBind) {
     const char *operation = isBind ? "bind" : "unbind";
     int index = invalidIndex;
 
+    if (this->drm.isVmBindSystemAlloc()) {
+         drm_xe_vm_bind bind = {};
```

Comment:
why this path is taken for all vm_binds?
this overwrites other ioctl calls, please unify the logic so we have a single ioctl calling point